amundsen_application/static/js/config/config-types.ts

```diff
@@ -77,6 +79,16 @@ interface BadgeConfig {
   [badge: string]: BadgeStyleConfig;
 }
 
+/**
+ * DateConfig - Configure various date formats
+ *
+ */
+interface DateFormatConfig {
```

Comment:
We can table this idea for if we ever dedicate ourselves to internationalization, but what I think this would ideally look like at scale would be to consider writing code leveraging [Intl.DataTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat/format) and its various options. Then users would simply configure their country code like `en-US`, which would get applied in that logic.